meson.build

```diff
@@ -198,6 +198,11 @@ if get_option('enable-librom')
 
 endif
 
+# Eigen library for linear algebra
+if get_option('enable-eigen')
+  su2_deps += [declare_dependency(include_directories: 'externals/eigen')]
+endif
```

Comment:
We can add it as a submodule.

Reply:
Thanks for the suggestion.
I checked the other WIP projects and it seems that Ole uses Eigen in his draft  #1217.
In addition, I found that Eigen is already included in subprojects/Mutationpp/thirdparty/eigen.
I can ask this in the next developer meeting, maybe there is more interest in adding it as a submodule.